crates/services/gas_price_service/src/v1/service.rs

```diff
@@ -481,10 +506,12 @@ mod tests {
             ),
             None,
         );
+        let latest_gas_price = Arc::new(parking_lot::RwLock::new((0, 0)));
```

Comment:
curious why we need this when perhaps the algo could contain this internally?

Reply:
Yeah... That works too. It's kinda a separate value so I thought to keep it separate but that would work.

Reply:
Yeah, maybe we oversplitting here. Algorithm could answer this, if we store the last used gas price there. But I think Mitch wants to reuse the code for v0. But maybe it can live in common and used in both algorithm.